advisories/unreviewed/2024/03/GHSA-jm46-725r-hh9v/GHSA-jm46-725r-hh9v.json

```diff
@@ -6,15 +6,31 @@
   "aliases": [
     "CVE-2024-0450"
   ],
-  "details": "An issue was found in the CPython `zipfile` module affecting versions 3.12.2, 3.11.8, 3.10.13, 3.9.18, and 3.8.18 and prior.\n\nThe zipfile module is vulnerable to “quoted-overlap” zip-bombs which exploit the zip format to create a zip-bomb with a high compression ratio. The fixed versions of CPython makes the zipfile module reject zip archives which overlap entries in the archive.\n\n",
+  "summary": "CPython zipfile module vulnerable to \"quoted-overlap\" zip-bombs",
+  "details": "An issue was found in the CPython `zipfile` module affecting versions 3.12.1, 3.11.7, 3.10.13, 3.9.18, and 3.8.18 and prior.\n\nFixed in 3.12.2, 3.11.8.\n\nThe zipfile module is vulnerable to “quoted-overlap” zip-bombs which exploit the zip format to create a zip-bomb with a high compression ratio. The fixed versions of CPython makes the zipfile module reject zip archives which overlap entries in the archive.\n\n",
   "severity": [
     {
       "type": "CVSS_V3",
       "score": "CVSS:3.1/AV:L/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
     }
   ],
   "affected": [
-
+    {
+      "package": {
+        "ecosystem": "PyPI",
+        "name": ""
+      },
+      "ranges": [
+        {
+          "type": "ECOSYSTEM",
+          "events": [
+            {
+              "introduced": "0"
+            }
+          ]
+        }
+      ]
+    }
```

Comment:
unfortunately it seems I cannot edit this auto-created branch, so I'll put it here as a suggestion instead:
```suggestion

```
Removing the 'affected' details which I was forced to add (as explained [here](https://github.com/github/advisory-database/pull/4204#issuecomment-2034168354))